libkbfs/bsplitter_simple.go

```diff
@@ -74,14 +74,18 @@ func NewBlockSplitterSimple(desiredBlockSize int64,
 			"desired size of %d", desiredBlockSize)
 	}
 
+	// Trial and error shows that this magic 75% constant maximizes
+	// the number of realistic indirect pointers you can fit into the
+	// default block size.  TODO: calculate this number more exactly
+	// during initialization for a given `maxSize`.
```

Comment:
Well `bpSize` is defined using `reflect` on the `BlockPointer` structure, it doesn't actually take serialization overhead into account (or `omitempty`, or anything like that).